metadata_service/__init__.py

```diff
@@ -60,6 +61,7 @@ def create_app(*, config_module_class: str) -> Flask:
 
     else:
         app = Flask(__name__)
+        CORS(app)
```

Comment:
Based on the documentation, it seems some security measure needs to be in place prior to enable `CORS`.
https://flask-cors.readthedocs.io/en/latest/

How about taking environment variable to enable this feature and let's disable it by default. What do you think?

Reply:
Sounds fair enough.
I think by the end of the week I will get to this MR and this [one](https://github.com/lyft/amundsensearchlibrary/pull/89)